_templates/collection/new/add-collection-export.ejs.t

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Remove collection template. It's a premature optimization that needs more thinking